packages/syft/src/syft/oblv/model.py

```diff
@@ -0,0 +1,441 @@
+            self.user_key_name,
+            self.user_key_name + "_public.der",
+        )
+        private_file_name = os.path.join(
+            os.path.expanduser("~"),
+            ".ssh",
+            self.user_key_name,
+            self.user_key_name + "_private.der",
+        )
+        log_file_name = os.path.join(
+            os.path.expanduser("~"),
+            ".oblv_syft_logs",
+            "proxy_logs_" + datetime.now().strftime("%d_%m_%Y_%H_%M_%S") + ".log",
+        )
+        # Creating directory if not exist
+        os.makedirs(os.path.dirname(log_file_name), exist_ok=True)
+        log_file = open(log_file_name, "wb")
+        depl = self.oblv_client.deployment_info(self.deployment_id)
+        if depl.is_deleted:
+            raise Exception(
+                "User cannot connect to this deployment, as it is no longer available."
+            )
+        try:
+            if depl.is_dev_env:
+                process = subprocess.Popen(
+                    [
+                        "oblv",
+                        "connect",
+                        "--private-key",
+                        private_file_name,
+                        "--public-key",
+                        public_file_name,
+                        "--url",
+                        depl.instance.service_url,
+                        "--pcr0",
+                        depl.pcr_codes[0],
+                        "--pcr1",
+                        depl.pcr_codes[1],
+                        "--pcr2",
+                        depl.pcr_codes[2],
+                        "--port",
+                        "443",
+                        "--lport",
+                        str(connection_port),
+                        "--disable-pcr-check",
+                    ],
+                    stdout=log_file,
+                    stderr=log_file,
+                )
+            else:
+                process = subprocess.Popen(
+                    [
+                        "oblv",
+                        "connect",
+                        "--private-key",
+                        private_file_name,
+                        "--public-key",
+                        public_file_name,
+                        "--url",
+                        depl.instance.service_url,
+                        "--pcr0",
+                        depl.pcr_codes[0],
+                        "--pcr1",
+                        depl.pcr_codes[1],
+                        "--pcr2",
+                        depl.pcr_codes[2],
+                        "--port",
+                        "443",
+                        "--lport",
+                        str(connection_port),
+                    ],
+                    stdout=log_file,
+                    stderr=log_file,
+                )
+            log_file_read = open(log_file_name, "r")
+            while True:
+                d = log_file_read.readline()
+                if d.__contains__("Error:  Invalid PCR Values"):
```

Comment:
OUGHTTO + Q: Why not `find`?

Reply:
Solved in another local mode PR by [7536c37](https://github.com/OpenMined/PySyft/commit/7536c377339ddfb39c9a17253b7998a42def1dcc)